src/main/java/EchoBot.java

```diff
@@ -0,0 +1,92 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.FindCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.ui.Ui;
```

Comment:
I like how you import the classes individually! 👍

Reply:
Maybe you can consider renaming your package to `echobot` instead of `duke` so that it is standardised?